File: U3P03-Acceso al catalogo con PHP/catalogo/Obra.php
<?php 
class Obra{
    private $idObra;
    private $idAutor;
    private $nombre;
    private $imagenObra;
    private $nomAutor;
    private $imagenAutor;
    
    public function getIdObra(){
        return $this->idObra;
    }
    public function getIdAutor(){
        return $this->idAutor;
    }
    public function getNombre(){
        return $this->nombre;
    }
    
    public function getImagenObra(){
        return "<img src='/U3P03-Acceso%20al%20catalogo%20con%20PHP/img/".$this->imagenObra."'height=200px width=200px>";
    }
    
    public function getNomAutor(){
        return $this->nomAutor;
    }

    public function getImagenAutor(){
        return "<img src='/U3P03-Acceso%20al%20catalogo%20con%20PHP/img/".$this->imagenAutor."'height=200px width=200px>";
    }
}
?>

File: U5P02-Java-Catalogo/src/servlets/Obra.java
package servlets;
public class Obra {
	private String nombre, imagenObra, nomAutor, imagenAutor;
	private int idObra, idAutor;
	
	public Obra(String nombre, String imagenObra, String nomAutor, String imagenAutor, int idObra, int idAutor) {
		this.nombre=nombre;
		this.imagenObra=imagenObra;
		this.nomAutor=nomAutor;
		this.imagenAutor=imagenAutor;
		this.idObra=idObra;
		this.idAutor=idAutor;
	}
	
	public String getNombre() {return nombre;}
	public String getImagenObra() { return "<img src='/U5P02-Java-Catalogo/img/"+imagenObra+"'height=200px width=200px>";}
	public String getNomAutor() {return nomAutor;}
	public String getImagenAutor() {return "<img src='/U5P02-Java-Catalogo/img/"+imagenAutor+"'height=200px width=200px>";}
	public int getIdObra() {return idObra;}
	public int getIdAutor() {return idAutor;}

}